docs/examples/kops-test-route53-subdomain.md

```diff
@@ -0,0 +1,870 @@
+||+---------------------------------------------------------------------------------+||
+|||  ns-1383.awsdns-44.org.                                                         |||
+|||  ns-829.awsdns-39.net.                                                          |||
+|||  ns-346.awsdns-43.com.                                                          |||
+|||  ns-1973.awsdns-54.co.uk.                                                       |||
+||+---------------------------------------------------------------------------------+||
+||                                ResourceRecordSets                                 ||
+|+-------------------------------------------------------+------------+--------------+|
+||                         Name                          |    TTL     |    Type      ||
+|+-------------------------------------------------------+------------+--------------+|
+||  kopsclustertest.kopeio.org.                          |  900       |  SOA         ||
+|+-------------------------------------------------------+------------+--------------+|
+|||                                 ResourceRecords                                 |||
+||+---------------------------------------------------------------------------------+||
+|||                                      Value                                      |||
+||+---------------------------------------------------------------------------------+||
+|||  ns-1383.awsdns-44.org. awsdns-hostmaster.amazon.com. 1 7200 900 1209600 86400  |||
+||+---------------------------------------------------------------------------------+||
+```
+
+Also, do a "dig" test in order to check the zone availability on the Internet:
+
+```bash
+dig +short kopsclustertest.kopeio.org soa
+
+ns-1383.awsdns-44.org. awsdns-hostmaster.amazon.com. 1 7200 900 1209600 86400
+
+dig +short kopsclustertest.kopeio.org ns
+
+ns-1383.awsdns-44.org.
+ns-829.awsdns-39.net.
+ns-1973.awsdns-54.co.uk.
+ns-346.awsdns-43.com.
+```
+
+If both your SOA and NS records are there, then your subdomain is ready to be used by KOPS.
+
+
+## AWS/KOPS ENVIRONMENT INFORMATION SETUP:
+
+First, using some scripting and asuming you already configured your "aws" environment on your linux system, use the following commands in order to export your AWS access/secret (this will work if you are using the default profile):
+
+```bash
+export AWS_ACCESS_KEY_ID=`grep aws_access_key_id ~/.aws/credentials|awk '{print $3}'`
+export AWS_SECRET_ACCESS_KEY=`grep aws_secret_access_key ~/.aws/credentials|awk '{print $3}'`
+echo "$AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY"
+```
+
+If you are using multiple profiles (and not the default one), you should use the following command instead in order to export your profile:
+
+```bash
+export AWS_PROFILE=name_of_your_profile
+```
+
+Create a bucket (if you don't already have one) for your cluster state:
+
+```bash
+aws s3api create-bucket --bucket my-kops-s3-bucket-for-cluster-state --region us-east-1
```

Comment:
Can we turn on versioning?

Reply:
We usually recommend to in life the base domain the the bucket name kops-example.org ~ something like that

Reply:
Good idea. Will do !